pipeline.Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:3.6
+
+ENV DOCKERVERSION=18.03.1-ce
+
+RUN curl -fsSLO https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKERVERSION}.tgz \
+  && tar xzvf docker-${DOCKERVERSION}.tgz --strip 1 \
+                 -C /usr/local/bin docker/docker \
+  && rm docker-${DOCKERVERSION}.tgz
+
+RUN apt-get update && apt-get install -y software-properties-common \
+	&& echo 'deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main' >> /etc/apt/sources.list \
+	&& apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 93C4A3FD7BB9C367 \
+	&& apt-get update \
+	&& curl -sL https://deb.nodesource.com/setup_8.x | bash - \
```

Comment:
please remove this as was agreed

Reply:
Docker and nodejs removed,
Regarding ansible repo - it's ok